renderdoc/driver/d3d11/d3d11_context1_wrap.cpp

```diff
@@ -460,7 +460,15 @@ bool WrappedID3D11DeviceContext::Serialise_CopySubresourceRegion1(
       if(pDstResource && pSrcResource)
       {
         draw.copySource = srcOrigID;
+        draw.copySourceSubresource = Subresource(
+            GetMipForSubresource(GetResourceManager()->UnwrapResource(pSrcResource), SrcSubresource),
```

Comment:
There's no need to unwrap before calling `Get*ForSubresource` functions, which should make this a little more compact and easy to read.

Reply:
sounds good. I'll get back on this later in the week (or over the weekend).

Reply:
Done. re-tested and rebased on latest.